frontend/packages/cli/src/cli/initCommand/index.ts

```diff
@@ -1,20 +1,132 @@
+import fs from 'node:fs'
+import path from 'node:path'
 import { Command } from 'commander'
-import { bold, green } from 'yoctocolors'
+import inquirer from 'inquirer'
+import * as yocto from 'yoctocolors'
 
 const initCommand = new Command('init').description(
   'guide you interactively through the setup',
 )
 
-initCommand.action(() => {
-  console.info(
-    green(
-      bold(`
-    👾  Welcome to the @liam-hq/cli setup process! 👾
+const formatMap: Record<string, string> = {
+  PostgreSQL: 'postgresql',
+  'Ruby on Rails (schema.rb)': 'schemarb',
+  Prisma: 'prisma',
+  // Drizzle: 'drizzle',
+}
+
+initCommand.action(async () => {
+  console.info(`
+👾  Welcome to the @liam-hq/cli setup process! 👾
+
+This \`init\` subcommand will guide you interactively through the setup.
+
+${yocto.greenBright('🌟 This init command is a work in progress! 🌟')}
+We’re continuously improving it. Don’t forget to run \`npx @liam-hq/cli init\` after updates for the latest features.
+
+💡 Have feedback? Share it with us!  
+Visit ${yocto.blueBright('https://github.com/liam-hq/liam/discussions')} to submit ideas or report issues.
+
+🌟️ ${yocto.bold('Love Liam ERD')}? Help us grow by starring our GitHub repository:  
+${yocto.blueBright('https://github.com/liam-hq/liam')}
+
+----
+
+Now, let’s get started with setting up your Liam ERD project.
+  `)
+
+  const answers = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'dbOrOrm',
+      message: 'Which Technology (database or ORM) are you using?',
+      choices: [
+        'PostgreSQL',
+        'Ruby on Rails (schema.rb)',
+        'Prisma (schema.prisma)',
+        // 'Drizzle',
+      ],
+      default: 'PostgreSQL',
+    },
+    {
+      type: 'confirm',
+      name: 'canUsePgDump',
+      message:
+        'Can we use an .sql file generated by pg_dump? (Recommended: `pg_dump --schema-only`)',
+      default: false,
+      when: (prev) => prev.dbOrOrm === 'PostgreSQL',
+    },
+    {
+      type: 'confirm',
+      name: 'addGhActions',
+      message: 'Generate GitHub Actions Workflow?',
+      default: false,
+    },
+  ])
+
+  const selectedFormat = formatMap[answers.dbOrOrm] || 'postgresql'
+
+  console.info(`
+For more details about Liam ERD usage and advanced configurations, check out:
+${yocto.blueBright('https://liambx.com/docs')}
+`)
+
+  console.info('\n--- Output ---')
```

Comment:
Looks like this line doesn't need to be there.

Reply:
It does seem like this line doesn't need to be there. I agree it feels fine to remove it for now."